mewarpx/mewarpx/diags_store/checkpoint_diagnostic.py

```diff
@@ -61,35 +62,38 @@ def add_checkpoint(self):
 
         mwxrun.simulation.add_diagnostic(diagnostic)
 
+    def regular_checkpoint_manager(self):
+        """Function to handle checkpoints dumped on regular timesteps (not
+        during a SIGTERM event)."""
```

Comment:
Probably best not to 'hard-code' what signals might set the termination flag that it checks.